docker-compose-local.yml

```
@@ -0,0 +1,87 @@
+    container_name: ten-db
+    image: postgres:latest
+    ports:
+      - "5433:5432"
+    environment:
+      POSTGRES_USER: sa
+      POSTGRES_PASSWORD: sa
+      TZ: Asia/Seoul
+    volumes:
+      - ./init.sql:/docker-entrypoint-initdb.d/init.sql
+      - ./postgres-data:/var/lib/postgresql/data
+    command: [ "postgres", "-c", "log_statement=all", "-c", "log_destination=stderr" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)